workspaceitems.md

```diff
@@ -121,6 +121,21 @@ In addition, it allows in future to change the 1:1 association between collectio
 
 It returns the workspaceitem created by the specified submitter
 
+## Get Single Workspace Item from Item UUID
+
+**/api/submission/workspaceitems/item/<:uuid>**
```

Comment:
please move in the search methods area and use the standard url structure
`/api/submission/workspaceitems/search/item?uuid=<:item-uuid>`

Reply:
This again could alternatively be resolvable via a redirect. See this comment https://github.com/DSpace/Rest7Contract/pull/97#discussion_r376538335

Reply:
according to https://github.com/DSpace/Rest7Contract/pull/97#discussion_r377561234 as we already have search method that return single object I still recommend a search method here

Reply:
Similar to https://github.com/DSpace/Rest7Contract/pull/97#discussion_r378101428, this is a find method returning one workspace item.
I will adjust it as such in the contract